Feature/galactic support (#18)
eProsima/ROS2-SH: README.md

  ~/is_ws$ colcon build --cmake-args -DBUILD_ROS2_TESTS=ON
  ```

> * `IS_ROS2_VERSION`: Users can set this flag to any of the supported *ROS 2* distros to date to choose
>   which one should be used to compile the *ROS 2 System Handle*.
>   If not set, the *ROS 2* distro version will be retrieved from the last sourced *ROS distro* in the
>   compilation environment, this meaning that if the last sourced *ROS* environment was part of *ROS 1*,
>   the compilation process will stop and warn the user about it.

Reviewer: ```suggestion
* `IS_ROS2_VERSION`: This flag is intended to select the *ROS 2* distro that should be used to compile the *ROS 2 System Handle*. If not set, the version will be retrieved from the last *ROS distro* sourced in the compilation environment, this means that if the last *ROS* environment sourced corresponds to *ROS 1*, the compilation process will stop and warn the user about it.
```
